applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    cbar = plt.colorbar(c, ax=ax, orientation='horizontal', pad=0.1, shrink=0.8)
</pre><p>I know this is probably just a copy/paste from geocat-examples, but I think we should avoid unnecessary formatting stuff like colorbar orientation, pad, and shrink.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I added shrink and pad to make the colorbar not overlap with the longitude tick labels.</p>